static/js/formatters-internal.js

```diff
@@ -540,20 +541,20 @@ export function priceRange(defaultPriceRange, countryCode) {
  *                                          highlight.
  */
 export function highlightField(fieldValue, matchedSubstrings = []) {
-  let highlightedString = fieldValue;
+  let highlightedString = '';
 
   // We must first sort the matchedSubstrings by decreasing offset. 
```

Comment:
This comment is out of date now that we are sorting by ascending offset

Reply:
could we get some unit tests as well? specifically for cases with multiple highlighted substrings, since those are a little tricky

Reply:
yup, just added!